apps/hash-graph/docker/Dockerfile

```diff
@@ -5,22 +5,29 @@ ENV RUSTUP_HOME=/usr/local/rustup \
     PATH=:$PATH:/usr/local/cargo/bin
 
 COPY rust-toolchain.toml .
-RUN apk add --no-cache gcc musl-dev && \
+RUN apk add --no-cache gcc musl-dev bash && \
     wget -q -O- https://sh.rustup.rs | sh -s -- -y --default-toolchain none --profile minimal && \
     rustup show
 
+SHELL ["bash", "-c"]
```

Comment:
This is required for the check, which is used in `builder`

Reply:
How come this afect the next stage of the dockerfile? Shouldn't this call be under line 15 for it to influence anything?

Reply:
We reuse this image in the next stage, so this propagates.